.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] You should have a test.md file with the text "This file should be ignored by git" and it should be ignored by git and is not present in your GitHub repository. Please implement it